workers/dockerfiles/Dockerfile.transcriptome

```diff
@@ -30,6 +30,10 @@ RUN cd RSEM && make install
 RUN rm -rf RSEM
 
 # Install Salmon
+
+# Tximport requires all experiments to be processed with the same version of Salmon to work https://github.com/AlexsLemonade/refinebio/issues/1496
+# This is something that should be considered when updating salmon, because
+# all unprocessed experiments will require to have salmon run on them again.
```

Comment:
```suggestion
# all samples from incomplete experiments must have salmon run on them again.
```
I think this is true with the suggested revision. If an experiment is not yet tximported, but all the samples are done, I think it will be fine. However, the current wording suggests that will also fail. Just a minor change for those who find this in the future.